packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -2,10 +2,19 @@ import React from 'react';
 import { bool, shape, string } from 'prop-types';
 
 import { useScrollLock } from '@magento/peregrine';
+import { useMiniCart } from '@magento/peregrine/lib/talons/MiniCart/useMiniCart';
 import { mergeClasses } from '@magento/venia-ui/lib/classify';
 
+import ProductListing from './ProductListing';
+
+import ShoppingBadOperations from './miniCart.gql';
```

Comment:
How bad are we talking about here? 😂

Reply:
Man, I should utilize the shutdown to improve my spellings, @sirugh pointed out [something similar](https://github.com/magento/pwa-studio/pull/2506#discussion_r445246028) earlier today. 🤣